examples/simple/webpack.config.js

```diff
@@ -36,6 +36,9 @@ module.exports = {
             'OptionTextElement',
             'RedirectionSideEffect',
             'RefreshSideEffect',
+            'AdminUIProps',
```

Comment:
We should spend some time upgrading babel, this is forced by a bug that was fixed a few months ago